code/go/0chain.net/smartcontract/storagesc/challenge_eventdb.go

```diff
@@ -0,0 +1,112 @@
+package storagesc
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+
+	"0chain.net/core/common"
+
+	"0chain.net/smartcontract/dbs"
+
+	cstate "0chain.net/chaincore/chain/state"
+	"0chain.net/smartcontract/dbs/event"
+)
+
+func storageChallengeToChallengeTable(ch *StorageChallengeInfo) *event.Challenge {
+	var validatorsArr []string
+	for _, v := range ch.Validators {
+		validatorsArr = append(validatorsArr, v.ID)
+	}
+	validators := strings.Join(validatorsArr, ",")
+	return &event.Challenge{
+		ChallengeID:    ch.ID,
+		CreatedAt:      ch.Created,
+		AllocationID:   ch.AllocationID,
+		BlobberID:      ch.BlobberID,
+		ValidatorsID:   validators,
```

Comment:
do we ever query "all challenges for current validator" or similar? if yes then we might normalize this field

Reply:
nope